server/channels/app/oembed/providers.go

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
It's good practice to use a `_gen.go` suffix for generated code. It would also make it consistent with other files we generate (e.g. https://github.com/mattermost/mattermost/blob/master/server/public/model/user_serial_gen.go and the like)